dll/win32/browseui/CAutoComplete.cpp

```diff
@@ -21,6 +21,7 @@
  */
 
 #include "precomp.h"
+#include <imm.h>
```

Comment:
Is this for `WM_IME_NOTIFY` or `IMN_OPENCANDIDATE` defines? (maybe add a `// For IMN_OPENCANDIDATE` or similar comment, if you feel like it)

Reply:
```suggestion
#include <imm.h> // IMN_OPENCANDIDATE
```